<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1300</width>
      <height>800</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="thuchi1Button">
         <property name="minimumSize">
          <size>
           <width>1</width>
           <height>1</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Chalkboard SE</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 255, 26);</string>
         </property>
         <property name="text">
          <string>Chi Tiêu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="xuatchau1Button">
         <property name="font">
          <font>
           <family>Chalkboard SE</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Xuất Chậu</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>70</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tổng Chi</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>361</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Nhập thông tin</string>
      </property>
      <widget class="QPushButton" name="muaBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>200</y>
         <width>261</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Thêm</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>32</y>
         <width>29</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tên:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>24</x>
         <y>120</y>
         <width>58</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Số Tiền: </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>80</y>
         <width>67</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Số Lượng:</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="tenmua">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>256</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="sotienmua">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>256</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="soluongmua">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>36</width>
         <height>21</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Chalkboard SE</family>
        <pointsize>23</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="displayFormat">
       <string>d.M.yyyy</string>
      </property>
     </widget>
     <widget class="QListWidget" name="tongchiList">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>100</y>
        <width>351</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="thuchi2Button">
         <property name="font">
          <font>
           <family>Chalkboard SE</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Chi Tiêu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="xuatchau2Button">
         <property name="font">
          <font>
           <family>Chalkboard SE</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 255, 11);</string>
         </property>
         <property name="text">
          <string>Xuất Chậu</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>381</width>
        <height>451</height>
       </rect>
      </property>
      <property name="title">
       <string>Nhập thông tin</string>
      </property>
      <widget class="QComboBox" name="tenChau">
       <property name="geometry">
        <rect>
         <x>125</x>
         <y>30</y>
         <width>248</width>
         <height>31</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Chậu Tròn 50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Chậu Tròn 60</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Chậu Tròn 70</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Chậu Tròn 80</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Chậu Tròn 90</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Chậu Tròn 100</string>
        </property>
       </item>
      </widget>
      <widget class="QSpinBox" name="soLuongXuatChau">
       <property name="geometry">
        <rect>
         <x>128</x>
         <y>77</y>
         <width>36</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="xuatchauBtn">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>400</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Chalkboard SE</family>
         <pointsize>18</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Thêm</string>
       </property>
      </widget>
      <widget class="QLabel" name="thanhtien">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>371</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>15</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Thành Tiền</string>
       </property>
      </widget>
      <widget class="QLabel" name="tongtien">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>190</y>
         <width>108</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Chalkboard SE</family>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>VND</string>
       </property>
      </widget>
      <widget class="QLabel" name="thanhtien_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>320</y>
         <width>361</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>15</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Thành Tiền</string>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Chalkboard SE</family>
            <pointsize>18</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Tên Chậu:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Chalkboard SE</family>
            <pointsize>18</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Số Lượng:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>Chalkboard SE</family>
            <pointsize>18</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Người Mua: </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>181</y>
         <width>108</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Chalkboard SE</family>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tổng Tiền: </string>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>361</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <family>Chalkboard SE</family>
            <pointsize>18</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Đã Thanh Toán:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="daTra"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>75</width>
         <height>39</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="tueanh">
          <property name="text">
           <string>Tuệ Anh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="khac">
          <property name="text">
           <string>Khác</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>60</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tổng kết</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Chalkboard SE</family>
        <pointsize>23</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="displayFormat">
       <string>d.M.yyyy</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listXuatChau">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>100</y>
        <width>351</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
